tests/test_utils.py

```diff
@@ -0,0 +1,59 @@
+from asati.utils import parse_accession_number, parse_extent_data
+
+
+def test_parse_accession_number_4_parts_success():
+    accession_record = {
+        "id_0": "2025",
+        "id_1": "214",
+        "id_2": "314",
+        "id_3": "514",
+    }
+    assert parse_accession_number(accession_record) == "2025-214-314-514"
+
+
+def test_parse_accession_number_1_part_success():
+    assert parse_accession_number({"id_0": "2025"}) == "2025"
+
+
+def test_parse_extent_data_no_extents_success():
+    assert parse_extent_data({}) == {}
```

Comment:
These are nice tests.  I had some initial questions about the terse nature of the utility function, but these answered them!